lib/dune

```diff
@@ -0,0 +1,32 @@
+(* -*- tuareg -*- *)
+
+let coverage_rewriter ~full =
+  let is_coverage =
+    try
+      Jbuild_plugin.V1.run_and_read_lines "printenv BISECT_ENABLE" = ["YES"]
+    with _ -> false
+  in
+  match is_coverage, full with
+  | true, true -> "(preprocess (pps (bisect_ppx -conditional)))"
```

Comment:
I think this can be simplified to:
```
let coverage_rewriter =
  match Sys.getenv "BISECT_ENABLE" with
  | "YES" -> "bisect_ppx"
  | _ -> ""
  | exception Not_found -> ""
```

Nesting preprocess inside preprocess won't work anyway.

Reply:
We use this same boilerplate everywhere, that's why it's more general than necessary - `full` has to be true, if there are no other preprocessors, but only bisect. In that case, when bisect is disabled, the whole `(preprocess ...)` part has to be omitted.

Reply:
Fair enough, I was thinking that one could use `(preprocess (pps (%s))` in that case but not sure whether that would still build and run a ppx driver or not, so probably best to avoid preprocessing at all if not needed for better build performance.